src/js/state/reducers/pageData.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const initState = {
+  fragments: [],
+  url: '',
```

Comment:
Should this rather be dictionary? As it seems to me, user can open second devtools on different page and it will override data for first one, i guess this can be stored as `tabId: { fragment, url }` to avoid that